compiler/passes/scopeResolve.cpp

```diff
@@ -2311,6 +2366,37 @@ static Symbol* inSymbolTable(const char* name, BaseAST* ast) {
   return retval;
 }
 
+Symbol* lookupInModuleOrBuiltins(ModuleSymbol* mod, const char* name,
+                                 int& nSymbolsFound) {
+  BaseAST* scope = mod->block;
+  if (Symbol* sym = inSymbolTable(name, scope)) {
+    return sym;
```

Comment:
Why doesn't this also set `nSymbolsFound` like the other if statements in the function?

Reply:
Good point, I will adjust it.